integration/apidefinition_test.go

```diff
@@ -826,22 +826,22 @@ func TestApiDefinitionClientMTLS(t *testing.T) {
 	testWithCert := features.New("Client MTLS with certificate").
 		Setup(func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
 			client := envConf.Client()
-			is := is.New(t)
+			eval := is.New(t)
 			opConfSecret := v1.Secret{}
 
 			err := client.Resources(opNs).Get(ctx, "tyk-operator-conf", opNs, &opConfSecret)
-			is.NoErr(err)
+			eval.NoErr(err)
 
 			data, ok := opConfSecret.Data["TYK_AUTH"]
 			if !ok {
-				is.Fail()
+				eval.Fail()
 			}
```

Comment:
what do you think about using `eval.True()` instead of `if` statement?

```go
eval.True(ok)
```

Reply:
Makes complete sense. I will update it